### PR TITLE
Added Space Packages Publication

### PR DESCRIPTION
**Pull request by Mr3zee**
Publish with: `./gradlew publishMavenPublicationToSpaceRepository`
Setup:
- `SPACE_PACKAGES_USERNAME` or `kotlin.mcp.sdk.packages.username`
- `SPACE_PACKAGES_PASSWORD` or `kotlin.mcp.sdk.packages.password`
- `libs.sign.key.id` (optional, signing)
- `libs.sign.key.private` (optional, signing)
- `libs.sign.passphrase` (optional, signing)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,19 +17,19 @@ The Model Context Protocol allows applications to provide context for LLMs in a
 
 ## Installation
 
-Add the JitPack repository to your build file:
+Add the Space repository to your build file:
 
 ```kotlin
 repositories {
-    maven { url = uri("https://jitpack.io") }
+    maven { url = uri("https://maven.pkg.jetbrains.space/public/p/kotlin-mcp-sdk/sdk") }
```
Comment (e5l) [inline README.md:24]:
```suggestion
    maven(url="https://maven.pkg.jetbrains.space/public/p/kotlin-mcp-sdk/sdk")
```

File: README.md [hunk 2]
```diff
@@ -17,19 +17,19 @@ The Model Context Protocol allows applications to provide context for LLMs in a
...
 }
 ```
 
 Add the dependency:
 
 ```kotlin
 dependencies {
-    implementation("com.github.JetBrains:mcp-kotlin-sdk:main-SNAPSHOT")
+    implementation("com.github.JetBrains:mcp-kotlin-sdk:1.0-SNAPSHOT")
```
Comment (e5l) [inline README.md:32]: Should we make 0.1.0 the first release? 1.0 please avoid using SNAPSHOT in the version

File: README.md [hunk 3]
```diff
@@ -17,19 +17,19 @@ The Model Context Protocol allows applications to provide context for LLMs in a
 
 ## Installation
 
-Add the JitPack repository to your build file:
+Add the Space repository to your build file:
```
Comment (e5l) [inline README.md:20]:
```suggestion
Add the new repository to your build file:
```

File: build.gradle.kts [hunk 4]
```diff
@@ -7,6 +7,14 @@ plugins {
     `maven-publish`
 }
 
+if (project.getSensitiveProperty("libs.sign.key.private") != null) {
+    apply(plugin = "signing")
+}
+
+tasks.withType<PublishToMavenRepository>().configureEach {
```
Comment (e5l) [inline build.gradle.kts:14]: Should this dependency be inside if?

Comment (Mr3zee) [reply build.gradle.kts:14]: Doesn't matter

**Pull request merged**